### PR TITLE
LBSD-643 Enhance blog list link

### PR DESCRIPTION
**Pull request by vladnicoara**
The link to the list of blogs in theme manager must be bigger

### CODE REVIEW

**Comment by vied12** [comment]
Maybe you can still use one way binding here
/poke @plamut :ghost:

**Comment by nistormihai** [comment]
LGTM

File: client/protractor-conf.js [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 exports.config = {
     allScriptsTimeout: 30000,
-    baseUrl: 'http://localhost:9090',
+    baseUrl: 'http://localhost:9000',
```
Comment (vied12) [inline client/protractor-conf.js]: Please, don't :sweat_smile:

File: server/settings.py [hunk 2]
```diff
@@ -194,7 +194,7 @@ def env(variable, fallback_value=None):
 else:
     INSTALLED_APPS.append('apps.auth.db')
 
-SUPERDESK_TESTING = (env('SUPERDESK_TESTING', 'false').lower() == 'true')
+SUPERDESK_TESTING = (env('SUPERDESK_TESTING', 'true').lower() == 'true')
```
Comment (vied12) [inline server/settings.py]: Can I know why please?

File: client/protractor-conf.js (hunk 1 above)
Comment (vladnicoara) [inline client/protractor-conf.js]: wow, committed the wrong file, just use this locally

File: server/settings.py (hunk 2 above)
Comment (vladnicoara) [inline server/settings.py]: same here, this file went to the commit by mistake

**Comment by vied12** [comment]
When there is 0 blog, it shows `0 blogs`. It should be `0 blog`

**Comment by nistormihai** [comment]
@vied12 you see why gettext is needed for pluralization
PS: gettext is supporting up to 5 messages for plural form, in `fr` I thing there are 3 forms, and it is not a problem if `0 blogs` in `fr` can be `0 le blog`

File: client/spec/themes_manager_spec.js [hunk 3]
```diff
@@ -12,8 +12,6 @@ describe('Themes Manager', function() {
         .themes
         .then(function(themes) {
             expect(themes.length).toBe(3);
-            // angular-base
-            themeManager.expectTheme(0, {number_of_blogs_expected: 0, is_default_theme:false, name: 'Angular Base'});
```
Comment (vied12) [inline client/spec/themes_manager_spec.js]: We need to keep this check

**Pull request merged**